ui/lib/sync/addon/routes/secrets/destinations/index.ts

```diff
@@ -68,4 +82,14 @@ export default class SyncSecretsDestinationsIndexRoute extends Route {
       typeFilter: params.type,
     });
   }
+
+  resetController(controller: SyncSecretsDestinationsController, isExiting: boolean) {
+    if (isExiting) {
+      controller.setProperties({
+        page: undefined,
+        name: undefined,
+        type: undefined,
+      });
+    }
```

Comment:
This is necessary to reset the query param values since they are sticky by default.

Reply:
Should we also do this for `page` for the `secrets/destinations/destination/secrets` route?

Reply:
Good call I will update that as well